core/capabilities/triggers/logevent/service.go

```diff
@@ -0,0 +1,161 @@
+package logevent
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+)
+
+const ID = "log-event-trigger-%s-%d@1.0.0"
+
+const defaultSendChannelBufferSize = 1000
+
+// Log Event Trigger Capability Input
+type Input struct {
+}
+
+// Log Event Trigger Capability Payload
+type Payload struct {
+	// Time that Log Event Trigger's task execution occurred (RFC3339Nano formatted)
+	ActualExecutionTime string
+}
+
+// Log Event Trigger Capabilities Manager
+// Manages different log event triggers using an underlying triggerStore
+type TriggerService struct {
+	capabilities.CapabilityInfo
+	capabilities.Validator[RequestConfig, Input, capabilities.TriggerResponse]
+	lggr           logger.Logger
+	triggers       CapabilitiesStore[logEventTrigger, capabilities.TriggerResponse]
+	relayer        core.Relayer
+	logEventConfig Config
+	stopChan       services.StopChan
+	wg             sync.WaitGroup
+}
+
+// Common capability level config across all workflows
+type Config struct {
+	ChainID        uint64 `json:"chainId"`
```

Comment:
Should this be a chain-agnostic string chain ID?
```suggestion
	ChainID        string `json:"chainId"`
```

Reply:
Hmm.. sure
I'll change it to chain-agnostic string chain ID